### PR TITLE
TTT-1849_TTT-1874  created the swagger doc for version 1.0 of the API

### PR DESCRIPTION
**Pull request by claudiasion**
Please review the swagger doc for version 1.0

### CODE REVIEW

File: src/main/resources/travel-expedia-integration-api-v1.0.yaml [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+          type: string
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/ExpediaBatchResponsePayload'
+        '400':
+          description: Bad Request
+        '401':
+          description: Unauthorized
```
Comment (riccardoNovaglia) [inline src/main/resources/travel-expedia-integration-api-v1.0.yaml]: Are we going to have auth on this app?
Do we need to document these codes or do we just expect the app to return 200 for ok, 404 for not found, and some 500 for errors?

File: src/main/resources/travel-expedia-integration-api-v1.0.yaml [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
+  description: Api to handle Travel Expedia Integration
+  version: 1.0.0
+  title: Travel Expedia Integration API
+tags:
+  - name: Expedia
+    description: Expedia Integration
+schemes:
+  - http
+paths:
+  /batch:
```
Comment (riccardoNovaglia) [inline src/main/resources/travel-expedia-integration-api-v1.0.yaml]: I thought we would ask for a package by id (booking.com id maybe?)
Outside this app to integrate, nobody should know/care how requests to expedia are done, just how to ask for 1 package and what comes back from it. Maybe we need to revisit this

File: src/main/resources/travel-expedia-integration-api-v1.0.yaml (hunk 1 above)
Comment (claudiasion) [reply src/main/resources/travel-expedia-integration-api-v1.0.yaml]: Nope for now at least, I copied the format of the yaml and forgot to take out those response codes, thank you

File: src/main/resources/travel-expedia-integration-api-v1.0.yaml (hunk 2 above)
Comment (claudiasion) [reply src/main/resources/travel-expedia-integration-api-v1.0.yaml]: we were thinking for version 1.0 until more light is shed on how we are going build the calls to Expedia (what intervals, whether they are configured or sent to us, what hotels, where we get them from) to make a simple API that gets all this input (origin destination, hotels ids, time interval of search) and forwards it to Expedia with minor changes (adding the required headers and authentication) and we return what Expedia returns for now, without any mapping; let me if you want to setup a discussion regarding this today

File: src/main/resources/travel-expedia-integration-api-v1.0.yaml [hunk 3]
```diff
@@ -0,0 +1,420 @@
+swagger: '2.0'
+info:
+  description: Api to handle Travel Expedia Integration
```
Comment (adicontiu) [inline src/main/resources/travel-expedia-integration-api-v1.0.yaml]: The scope of this service is not Expedia only, but package ingestion, so we should modify the description, title etc.

File: src/main/resources/travel-expedia-integration-api-v1.0.yaml [hunk 4]
```diff
@@ -0,0 +1,420 @@
...
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '404':
+          description: Not Found
+definitions:
+  ExpediaBatchRequestPayload:
+    type: object
+    properties:
+      departureAirport:
```
Comment (adicontiu) [inline src/main/resources/travel-expedia-integration-api-v1.0.yaml]: The input should only contain hotel ids.

File: src/main/resources/travel-expedia-integration-api-v1.0.yaml (hunk 3 above)
Comment (claudiasion) [reply src/main/resources/travel-expedia-integration-api-v1.0.yaml]: The renaming of all: packages, classes, objects, files, settings is part of a different story which will be implemented as part of Sprint 2

**Comment by adicontiu** [comment]
We will have a different swagger interface.

**Pull request closed without merging**